views/employees/edit.php

```diff
@@ -28,7 +28,7 @@
     <label for="age">Employee age</label><br>
     <input type="number" name="age" step="1" min="15" max="100" value="<?php echo htmlspecialchars($filteredData['age']); ?>"><br><br>
 
-    <label for="sex">Employee sex</label><br>
+    <label for="sex">Employee sex (m/f)</label><br>
```

Comment:
If there are only two possibilites, it would be better to use `radio`s instead or `select`

Reply:
the issue has been fixed